testnet/stacks-node/src/tests/nakamoto_integrations.rs

```diff
@@ -167,13 +167,13 @@ lazy_static! {
         StacksEpoch {
             epoch_id: StacksEpochId::Epoch25,
             start_height: 201,
-            end_height: 231,
+            end_height: 251,
             block_limit: HELIUM_BLOCK_LIMIT_20.clone(),
             network_epoch: PEER_VERSION_EPOCH_2_5
         },
         StacksEpoch {
             epoch_id: StacksEpochId::Epoch30,
-            start_height: 231,
+            start_height: 251,
```

Comment:
I'm sorry I missed this earlier -- but is it possible to alter the test so that this change isn't necessary? I think add a parameter to ` SignerTest::new_with_conf...` that allows supplying a new epochs vec -- that way this change only gets applied in the new integration test and the other integration tests can still use 231.

Reply:
This change is definitely necessary for now otherwise we don't actually have an epoch 2.5 reward cycle with activated epoch 2.5 signers to sign any blocks as far as I can tell.

Reply:
It is possible that you merge your change and then I can do it, but until your change merges, I think its necessary.